upgrade/1.2/Stage_0_Prerequisites.md

```diff
@@ -158,146 +164,72 @@ ncn-m001# curl -H "Authorization: Bearer ${TOKEN}" -k -L -X POST 'https://api-gw
+- See the [Management Network User Guide](../../operations/network/management_network/index.md) for more information on the management network.
 
 <a name="prerequisites-check"></a>
+
 ## Stage 0.5 - Prerequisites Check
 
 Run check script:
 
-   Set the `SW_ADMIN_PASSWORD` environment variable to the admin password for the switches. This is needed for preflight tests within the check script.
+Set the `SW_ADMIN_PASSWORD` environment variable to the admin password for the switches. This is needed for preflight tests within the check script.
 
-   ```bash
-   ncn-m001# export SW_ADMIN_PASSWORD=sw1tCH@DM1Np4s5w0rd
-   ```
-
-   > **`IMPORTANT:`** If the password for the local Nexus `admin` account has
-   > been changed from the default `admin123` (not typical), then set the
-   > `NEXUS_PASSWORD` environment variable to the correct `admin` password
-   > before running prerequisites.sh!
-   >
-   > For example:
-   >
-   > ```bash
-   > ncn-m001# export NEXUS_PASSWORD=cu$t0m@DM1Np4s5w0rd
-   > ```
-   >
-   > Otherwise, a random 32-character base64-encoded string will be generated
-   > and updated as the default `admin` password when Nexus is upgraded.
-
-* Internet Connected
-
-    ```bash
-    ncn-m001# /usr/share/doc/csm/upgrade/1.2/scripts/upgrade/prerequisites.sh --csm-version [CSM_RELEASE] --endpoint [ENDPOINT]
-    ```
-
-    **NOTE** ENDPOINT is optional for internal use. It is pointing to internal arti by default.
-
-* Air Gapped
+```bash
+ncn-m001# export SW_ADMIN_PASSWORD=sw1tCH@DM1Np4s5w0rd
+```
 
-   ```bash
-   ncn-m001# /usr/share/doc/csm/upgrade/1.2/scripts/upgrade/prerequisites.sh --csm-version [CSM_RELEASE] --tarball-file [PATH_TO_CSM_TARBALL_FILE]
-   ```
+> **`IMPORTANT:`** If the password for the local Nexus `admin` account has
+> been changed from the default `admin123` (not typical), then set the
+> `NEXUS_PASSWORD` environment variable to the correct `admin` password
+> before running prerequisites.sh!
+>
+> For example:
+>
+> ```bash
+> ncn-m001# export NEXUS_PASSWORD=cu$t0m@DM1Np4s5w0rd
+> ```
+>
+> Otherwise, a random 32-character base64-encoded string will be generated
+> and updated as the default `admin` password when Nexus is upgraded.
+
+- ```bash
+  /usr/share/doc/csm/upgrade/1.2/scripts/upgrade/prerequisites.sh --csm-version [CSM_RELEASE]
+  ```
 
 **`IMPORTANT:`** If any errors are encountered, then potential fixes should be displayed where the error occurred. **IF** the upgrade `prerequisites.sh` script fails and does not provide guidance, then try rerunning it. If the failure persists, then open a support ticket for guidance before proceeding.
 
 **`IMPORTANT:`** If the `NEXUS_PASSWORD` environment variable was set as previously mentioned, then remove it before continuing:
 
-   ```bash
-   ncn-m001# export -n NEXUS_PASSWORD
-   ncn-m001# unset NEXUS_PASSWORD
-   ```
+```bash
+ncn-m001# export -n NEXUS_PASSWORD
+ncn-m001# unset NEXUS_PASSWORD
```

Comment:
I think the `export -n` is superfluous if you are going to unset it. Once it is unset, it is also no longer exported to new subprocesses, I believe.